code/game/objects/items/weapons/weaponry.dm

```diff
@@ -171,50 +171,64 @@
 	desc = "This thing looks dangerous... Dangerously good at baseball, that is."
 	homerun_able = 1
 
+/obj/item/melee/baseball_bat/badmin
+	name = "badmin bat"
+	desc = "SWING! SWING! SWING! SWING!"
+	icon_state = "baseball_bat_metal"
+	item_state = "baseball_bat_metal"
+
+/obj/item/melee/baseball_bat/badmin/can_deflect()
+	return TRUE // ALWAYS DEFLECT ALWAYS
+
 /obj/item/melee/baseball_bat/hit_reaction(mob/living/carbon/human/owner, attack_text, final_block_chance, damage, attack_type, atom/movable/AM)
 	. = ..()
-	if(!istype(AM, /obj/item) || attack_type != THROWN_PROJECTILE_ATTACK)
+	if(!istype(AM, /obj/item) || attack_type != THROWN_PROJECTILE_ATTACK && !istype(src, /obj/item/melee/baseball_bat/badmin))

```

Comment:
badmin bat doesn't care for your BULLETS

Reply:
Did you really.....*REALLY* do an `istype(src)` check.

Bad. And not just bad. *Terrible*

Reply:
It just ain't cricket

Reply:
AAAA